x/docs/md/cli/linux.md

```diff
@@ -1,7 +1,7 @@
 
 ## Linux
 
-View the CLI releases to [choose and download the appropriate package for your operating system](https://github.com/exercism/cli/releases/tag/v2.3.0).
+View the CLI releases to [choose and download the appropriate package for your operating system](https://github.com/exercism/cli/releases).
```

Comment:
Yes, I also think that this would be more correct but link text leads to *view __releases__* and *__choose__ the appropriate package*.

Maybe we should change the text as follows "View the latest CLI release to choose and download ..."?